jinahub/encoders/image/TimmImageEncoder/timm_encoder.py

```diff
@@ -0,0 +1,116 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Optional, List, Dict, Iterable, Tuple
+
+import numpy as np
+
+import torchvision.transforms as T
+import torch
+
+from jina import Executor, requests, DocumentArray
+from jina_commons.batching import get_docs_batch_generator
+
+from timm import create_model, list_models, list_modules
+from timm.data import resolve_data_config
+from timm.data.transforms_factory import create_transform
+
+
+class TimmImageEncoder(Executor):
+    """
+    :class:`TimmImageEncoder` encodes ``Document`` blobs of type `ndarray` (`float32`) and shape
+    `B x H x W x C` into `ndarray` of `B x D`.
```

Comment:
Information on the `B` dimension not needed. Our documents are individual images, the user does not need to know about the batch dimension. The executor does the batching internally

Reply:
Hi @tadejsv 
I was following ImageTorchEncoder for things like this. I will change this.